packages/bitcoin/src/helpers.tsx

```diff
@@ -29,3 +25,50 @@ export const getBalanceByMempool = async (address: string): Promise<Balance> =>
     throw new NoBalanceError();
   }
 };
+
+export const getInscriptionsByAddress = async ({
+  address,
+  size,
+  offset,
+}: {
+  address: string;
+  size: number;
+  offset: number;
+}): Promise<{ total: number; list: Inscription[] }> => {
+  const params = new URLSearchParams({ address, limit: `${size}`, offset: `${offset}` });
```

Comment:
似乎可以这样简写
```suggestion
  const params = new URLSearchParams({ address, limit: size, offset });
```

还有一个疑问，为啥要换成size，limit不是也挺常用的嘛，如果是limit的话这边改造成本更小了

```
export const getInscriptionsByAddress = async (params: InscriptionsByAddressParams): Promise<{ total: number; list: Inscription[] }> => { 

  const res = await fetch(`${HIRO_API}/ordinals/v1/inscriptions?${new URLSearchParams(params)}`);


```
![image](https://github.com/ant-design/ant-design-web3/assets/117748716/cf90c713-454e-417a-a26b-7b46a576bc78)

Reply:
URLSearchParams 传入要求 string, 分页参数是 number， 这里转化下
<img width="1035" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/115673583/d4da0944-bea4-4e9f-9d5b-b26f8901467b">


因为想和前面 Unisat 接口保持一致，不过把前面改回 limit 了，也没必要一致
<img width="372" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/115673583/bd6519fd-9b52-457c-ac61-99a544bd2313">